### ISSUE TITLE
[BUG] macOS Ventura 13.2 will not boot with Kryptonite

### ISSUE THREAD

**Issue by Piipperi** [mayankk2308/kryptonite#37]
### What Mac are you using?

MacBookPro11,3

### What version of macOS are you using?

Newer

### Are you using OpenCore Legacy Patcher?

Yes

### What is your issue?

It is possible that macOS 13.2 completely breaks Kryptonite. macOS 13.2 boots normally with just stock OpenCore Legacy Patcher, however, as soon as I put Kryptonite into the mix, it will not boot.

It seems to not be shown in any kind of logs. I had tried troubleshooting this during the beta period when I had the same issue, but I didn't bother writing a bug report in case it was just a case of bad install. So back then I just decided to leave my logs somewhere and downgraded back to 13.1. Last night, I updated to the official release of 13.2, with the machine not being able to boot.

No errors are shown in the logs, since the issue seems to be on macOS' side. The boot verbose log suggests it's able to reach the point where it's supposed to hand off the display to the eGPU, but the displays stay black and the verbose log is just stuck. The issue is also happening in Safe Mode, and for some odd reason, the laptop never shows up on my network (it's possible the Mac just completely freezes after trying to hand off the display output)

I will provide proper logs when I have time to troubleshoot, but for now I think this should be left as a fair warning.

### Upload Kryptonite Logs

Click here, then drag and drop your files in this area.

### Confirmation

- [X] I have uploaded the OpenCore **config.plist**.
- [X] I have uploaded all OpenCore Logs.
- [X] I have uploaded Lilu logs.
- [X] I have explained my issue in detail.

**Comment by Piipperi** [comment]
[Lilu_1.6.3_22.3.txt](https://github.com/mayankk2308/kryptonite/files/10492666/Lilu_1.6.3_22.3.txt)
[opencore-2023-01-24-172148.txt](https://github.com/mayankk2308/kryptonite/files/10492668/opencore-2023-01-24-172148.txt)
[config.txt](https://github.com/mayankk2308/kryptonite/files/10492670/config.txt)

**Comment by AshrafAnsari** [comment]
Yeah. Same issue here. Any version of macOS 13 works just fine with stock OpenCore Legacy Patcher but Kryptonite breaks it and won't boot. In other words, macOS 13 works fine on my 2013 MacPro with stock OpenCore, but can't get my eGPU to work.

**Comment by Piipperi** [comment]
> Yeah. Same issue here. Any version of macOS 13 works just fine with stock OpenCore Legacy Patcher but Kryptonite breaks it and won't boot. In other words, macOS 13 works fine on my 2013 MacPro with stock OpenCore, but can't get my eGPU to work.

This isn't the same issue. You have no chance of using Kryptonite on macOS 13 on a 6,1 Mac Pro unless you disable your FirePro GPUs.

**Comment by AshrafAnsari** [comment]
> > Yeah. Same issue here. Any version of macOS 13 works just fine with stock OpenCore Legacy Patcher but Kryptonite breaks it and won't boot. In other words, macOS 13 works fine on my 2013 MacPro with stock OpenCore, but can't get my eGPU to work.
> 
> This isn't the same issue. You have no chance of using Kryptonite on macOS 13 on a 6,1 Mac Pro unless you disable your FirePro GPUs.

Ok, I see. Are you saying there's a way to disable the FirePro GPUs?

**Comment by Piipperi** [comment]
There is a way to disable the FirePros in OpenCore config, but this will lead to no video output on macOS from the internal HDMI/Thunderbolt ports I think. I'd still advice against it even if it wasn't a problem, since it looks like macOS 13.2 is not compatible with Kryptonite anymore.

**Comment by cjhdc** [comment]
I also have this issue on my MacBook Pro 11,3. Has anyone attempted the Kryptonite install (with the updated Lilu kext drop in work-around) on 13.2.1?

**Comment by Albert--E** [comment]
Yes, on Monterey...I used the latest Lilu.kext. Never tried it on Ventura though.
